src/core/libmaven/mzUtils.cpp

```diff
@@ -502,59 +378,52 @@ Series:  Prentice-Hall Series in Automatic Computation
         if (stdA == 0 ) stdA=1.0;
         if (stdB == 0 ) stdB=1.0;
 
-        float t_test = (meanA-meanB)/sqrt(((stdA*stdA)/n1)+((stdB*stdB)/n2));
+        float t_test = (meanA - meanB) / sqrt(((stdA * stdA) / n1) +
+                                              ((stdB * stdB) / n2));
         return t_test;
     }
 
-
-    int countBelow(vector<float>& y, float ymax) {
-        vector<float>::iterator itr = lower_bound(y.begin(), y.end(), ymax);
-        int lb = itr-y.begin();
+   int countBelow(vector<float>& y, float ymax)
```

Comment:
This method is being used in only one place in the whole codebase. Can we move it to `CompareSamplesLogic` space itslef?